### PR TITLE
Clean up Xcode project

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

**Comment by marijnvdwerf** [comment]
@LRFLEW Anything I'm forgetting?

File: distribution/readme.txt [hunk 1]
```diff
@@ -69,7 +69,7 @@ following information in your bug report:
 3.0) Supported platforms
 ---- -------------------
 OpenRCT2 is currently supported on Windows Vista and above, many distributions of
-Linux and OSX 10.7. OpenRCT2 will only work on x86 architectures as the
+Linux and macOS 10.7. OpenRCT2 will only work on x86 architectures as the
```
Comment (pizza2004) [inline distribution/readme.txt]: Technically it's still OS X for older versions.  Only Sierra will be macOS, and I'm kind of wondering if they're even going to assign a number at all.

File: readme.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 An open source re-implementation of Roller Coaster Tycoon 2. A construction and management simulation video game that simulates amusement park management.
 
 ### Build Status
-|             | Windows | Linux / OSX | Download |
+|             | Windows | Linux / Mac | Download |
```
Comment (pizza2004) [inline readme.md]: I would say change the 3 instances of Mac OS X as headers to say just Mac, and the other instance of OS X to say macOS instead.

**Comment by marijnvdwerf** [comment]
@pizza2004 Care to go over the readme changes?

**Comment by pizza2004** [comment]
I think everything looks good to me now.  I'm glad to see everything get sorted and such.

**Comment by IntelOrca** [comment]
Can this be merged now?

**Comment by marijnvdwerf** [comment]
I think it can.

**Pull request merged**